plugins/processors/awsappsignals/factory.go

```diff
@@ -90,7 +90,7 @@ func createProcessor(
 ) (*awsappsignalsprocessor, error) {
```

Comment:
can we re-name our directory name to `awsapplicationsignals` as well?

Reply:
Done.